core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/json/JSONUtil.scala

```diff
@@ -0,0 +1,34 @@
+package edu.uci.ics.texera.workflow.operators.source.scan.json
+import com.fasterxml.jackson.databind.JsonNode
+
+import scala.jdk.CollectionConverters.asScalaIteratorConverter;
+
+object JSONUtil {
+  @throws[RuntimeException]
+  def parseJSON(
```

Comment:
Pls give a better name. You are taking JsonNode as input so I assume it's already parsed.

Reply:
changed it to `JSONToMap`